bundles/org.openhab.binding.tado/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -45,6 +45,9 @@
 			<channel typeId="timerDuration" id="timerDuration"></channel>
 
 			<channel typeId="operationMode" id="operationMode"></channel>
+			
+			<channel typeId="batteryLowAlarm" id="batteryLowAlarm"></channel>
```

Comment:
Ok, I will use the system type id. But it is only partially helpful since I need to add a binding specific description text to it (so the XML is not that much less).

Reply:
I decided finally NOT to use the system type id. The reason is that this channel must be read only, and the system type id is not so.

Reply:
I created a channel using the system.low-battery typeId and it's readonly for me. Can you check if it really isn't readonly for you. The reason we prefer system channels is because having standard channels makes it less work to translate channels, and also would make it more easily to handle similar channels.
This channel definition is the only thing left. If we can handle this we can merge this pr.

Reply:
Ok. Reverted.